cmd/admin-heal-status.go

```diff
@@ -0,0 +1,318 @@
+/*
+ * Minio Client (C) 2017 Minio, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cmd
+
+import (
+	"fmt"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"golang.org/x/tools/container/intsets"
+
+	humanize "github.com/dustin/go-humanize"
+	"github.com/minio/cli"
+	"github.com/minio/mc/pkg/console"
+	"github.com/minio/minio/pkg/madmin"
+)
+
+var adminHealStatusCmd = cli.Command{
+	Name:            "status",
+	Usage:           "Status of a heal task",
+	Before:          setGlobalsFromContext,
+	Action:          mainAdminHealStatus,
+	Flags:           globalFlags,
+	HideHelpCommand: true,
+	CustomHelpTemplate: `NAME:
+  {{.HelpName}} - {{.Usage}}
+
+USAGE:
+  {{.HelpName}} [FLAGS] TARGET
+
+FLAGS:
+  {{range .VisibleFlags}}{{.}}
+  {{end}}
+EXAMPLES:
+    1. To fetch status of a running heal task with no bucket and object-prefix in a Minio server with alias 'play'
+       $ {{.HelpName}} play
+
+    2. To fetch status of a running heal task on bucket 'testbucket' in a Minio server with alias 'play'
+       $ {{.HelpName}} play/testbucket/
+
```

Comment:
Remove the frivolous line.

Reply:
Are you sure it is not needed? I put it in there because every other command seems to be having a blank line at the end.